tb/core/interrupt_test/interrupt_test.c

```diff
@@ -515,8 +515,55 @@ void mat_mult(uint32_t mat1[MAT_DIM][MAT_DIM], uint32_t mat2[MAT_DIM][MAT_DIM],
     }
 }
 
+void activate_random_stall(void)
```

Comment:
Could we move this function into a separate file maybe? Now this function differs from the one used in https://github.com/davideschiavone/riscv/blob/improved_tb/tb/core/custom/hello_world.c (specifically the order of enabling is different).

If the Hello World program will be removed then you can ignore my comment.

Reply:
done in another PR